routing_common/bicycle_model.cpp

```diff
@@ -52,533 +51,533 @@ double constexpr kSpeedPierKMpH = 7.0;
-  { {"highway", "path"},           kSpeedPathKMpH },
-  { {"highway", "cycleway"},       kSpeedCyclewayKMpH },
-  { {"highway", "residential"},    kSpeedResidentialKMpH }, // *
-  { {"highway", "living_street"},  kSpeedLivingStreetKMpH }, // *
-  { {"highway", "steps"},          kSpeedStepsKMpH }, // *
-  { {"highway", "pedestrian"},     kSpeedPedestrianKMpH },
-  { {"highway", "platform"},       kSpeedPlatformKMpH }, // *
+  { {"highway", "primary"},        kSpeedPrimaryKMpH,       true },
+  { {"highway", "primary_link"},   kSpeedPrimaryLinkKMpH,   true },
+  { {"highway", "secondary"},      kSpeedSecondaryKMpH,     true },
+  { {"highway", "secondary_link"}, kSpeedSecondaryLinkKMpH, true },
+  { {"highway", "tertiary"},       kSpeedTertiaryKMpH,      true },
+  { {"highway", "tertiary_link"},  kSpeedTertiaryLinkKMpH,  true },
+  { {"highway", "service"},        kSpeedServiceKMpH,       true },
+  { {"highway", "unclassified"},   kSpeedUnclassifiedKMpH,  true },
+  { {"highway", "road"},           kSpeedRoadKMpH,          true },
+  { {"highway", "track"},          kSpeedTrackKMpH,         true },
+  { {"highway", "path"},           kSpeedPathKMpH,          true },
+  { {"highway", "cycleway"},       kSpeedCyclewayKMpH,      true },
+  { {"highway", "residential"},    kSpeedResidentialKMpH,   true },
+  { {"highway", "living_street"},  kSpeedLivingStreetKMpH,  true },
+  { {"highway", "steps"},          kSpeedStepsKMpH,         true },
+  { {"highway", "pedestrian"},     kSpeedPedestrianKMpH,    true },
+  { {"highway", "platform"},       kSpeedPlatformKMpH,      true },
 };
 
 // Brazil
 routing::VehicleModel::InitListT const g_bicycleLimitsBrazil =
 {
-  { {"highway", "trunk"},          kSpeedTrunkKMpH },
-  { {"highway", "trunk_link"},     kSpeedTrunkLinkKMpH },
-  { {"highway", "primary"},        kSpeedPrimaryKMpH },
-  { {"highway", "primary_link"},   kSpeedPrimaryLinkKMpH },
-  { {"highway", "secondary"},      kSpeedSecondaryKMpH },
-  { {"highway", "secondary_link"}, kSpeedSecondaryLinkKMpH },
-  { {"highway", "tertiary"},       kSpeedTertiaryKMpH },
-  { {"highway", "tertiary_link"},  kSpeedTertiaryLinkKMpH },
-  { {"highway", "service"},        kSpeedServiceKMpH }, // *
-  { {"highway", "unclassified"},   kSpeedUnclassifiedKMpH },
-  { {"highway", "road"},           kSpeedRoadKMpH },
-  { {"highway", "track"},          kSpeedTrackKMpH }, // *
-  { {"highway", "path"},           kSpeedPathKMpH },
-  { {"highway", "bridleway"},      kSpeedBridlewayKMpH },
-  { {"highway", "cycleway"},       kSpeedCyclewayKMpH },
-  { {"highway", "residential"},    kSpeedResidentialKMpH },
-  { {"highway", "living_street"},  kSpeedLivingStreetKMpH },
-  { {"highway", "steps"},          kSpeedStepsKMpH }, // *
-  { {"highway", "pedestrian"},     kSpeedPedestrianKMpH },
-  { {"highway", "footway"},        kSpeedFootwayKMpH },
-  { {"highway", "platform"},       kSpeedPlatformKMpH }, // *
+  { {"highway", "trunk"},          kSpeedTrunkKMpH,         true },
```

Comment:
Am I right that g_bicycleLimitsBrazil is the same as g_bicycleLimitsBelgium and the same as g_bicycleLimitsAustralia?